### DISCUSSION TITLE
New features of Prusa slicer 2.5

### DISCUSSION THREAD

**Discussion by ManuelPeiro** [supermerill/SuperSlicer#2926] [General]
hi,

I just saw that the version I think is alpha or beta of prusa 2.5, brings several interesting new features, which are:
- New slicer engine, Arachne. Improve small objects.
- Lighting infill
- Pressure equalizer, especially flexible filaments
- and Hide seams.

These news, which are still in alpha/beta in prusa slicer. Will they be incorporated into Super Slicer?
When? until the final version comes out? before?

Thank you

**Comment by supermerill** [comment]
> Will they be incorporated into Super Slicer?

yes

> When?

depends.

> until the final version comes out? 

before the full merge with the prusa 2.5 for some, yes.

**Reply by EvilGremlin** [reply]
Did you look at arachne code yet? Is it possible to combine with looping perimeters?

**Reply by supermerill** [reply]
yes, i looked at it.
For combining it with other things... it depends.
I guess looping perimeters can be added, as it's just after the perimeter creation.

**Reply by EvilGremlin** [reply]
Oh, good! Now we just need seam hiding on looping perimters (like Extra wipe?) and it'll be perfect! :)

**Comment by jowczarek85** [comment]
What about the new filament profiles? There are new Fiberlogy profiles in version 2.4 of PrusaSlicer :)

**Reply by supermerill** [reply]
I'll merge them with the 2.5 merge. In the meantime, you should be able to export them from prusa and import them into your superslicer.

**Comment by levleon13** [comment]
> * New slicer engine, Arachne. Improve small objects.

Cant wait for this !

**Comment by pka4916** [comment]
where did you see Hide Seams?   I am not finding that.
